src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2548,7 +2548,7 @@ class S4Importer final : public IParkImporter
 
                 ResearchItem tmpResearchItem = {};
                 ConvertResearchEntry(&tmpResearchItem, researchItem, researchType);
-                dst->Assoc = static_cast<uint32_t>(tmpResearchItem.rawValue);
+                dst->Assoc = tmpResearchItem.rawValue;
```

Comment:
Is this a conscious change to not copy the flags and category

Reply:
This hasn't changed, really. dst->Assoc is a 32-bit integer, it never fitted the category and now it also doesn't fit the flags. Only the research type (8 bit), entry index (16 bit) and ride type (8 bit) fit. Which is fine, since flags and category are not used in news items anyway.